CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master (unreleased)
 
 ### New features
+* [#7137](https://github.com/rubocop-hq/rubocop/issues/7137): Add new `Gemspec/RubyVersionGlobalsUsage` cop. ([@malyshkosergey][])
```

Comment:
I think that automatic merging doesn't work well because there is `### New features` in the master branch. Could you rebase on the latest master branch and squash your commits into one?